### ISSUE TITLE
[FEATURE] - Manual bookmarks sorting

### ISSUE THREAD

**Issue by lpsmith-dev** [alefragnani/vscode-bookmarks#585]
Add ability to manually sort the list of bookmarks in bookmarks view. Click and drag option to place a bookmark at any position of the list. It would be great that the tree view also supported manual sorting.

**Comment by manake** [comment]
I agree.

**Comment by vladrbyte** [comment]
line numbers in increasing / decreasing direction

**Comment by jaggujajamensan** [comment]
This or an option to sort them by label alphabetically when in list mode
When viewing in list mode the user has intentionally clicked away from tree-mode, so they wouldn't expect bookmark labels to still be sorted by the directory structure.

That way you could sort bookmarks by titling them, e.g. `[Defaults]`

**Comment by gianmarcoricciarelli** [comment]
I agree, it would be very nice to have such a feature, it would be very useful too.

**Comment by alefragnani** [comment]
Hi ,

With a new year about to start, it is the moment to review the extension’s backlog in order to keep it healthy and focused.

I'm closing this feature request as duplicate of #655 , as it shares most of it ideas.

Thanks for your understanding
